apps/sensenet/src/components/view-controls/permission-view.tsx

```diff
@@ -131,6 +132,25 @@ export const PermissionView: React.FC<PermissionViewProps> = (props) => {
     visitorEntries && visitorEntries.length > 0 ? setIsPrivate(false) : setIsPrivate(true)
   }, [permissions])
 
+  const selectMemberFunction = (newEntry: EntryType) => {
+    if (permissions) {
+      const localPermissions = { ...permissions }
```

Comment:
permissions.entries.push(newEntry) do the same, the permission.entries handled by reference in both cases